rslib/src/scheduler/states/load_balancer.rs

```diff
@@ -132,52 +144,74 @@ impl LoadBalancer {
         let (before_days, after_days) = constrained_fuzz_bounds(interval, minimum, maximum);
         let after_days = after_days + 1; // +1 to make the range inclusive of the actual value
 
-        // ok this looks weird but its a totally reasonable thing
-        // I want to be as close to the original interval as possible
-        // so this enumerates out from the center
-        // i.e. 0 -1 1 -2 2 .....
-        let intervals_to_check = (before_days..interval as u32)
-            .map(|before| before as i32 - interval as i32)
-            .chain(
-                (interval as u32..after_days)
-                    .filter(|i| *i >= minimum)
-                    .map(|after| after as i32 - interval as i32),
-            )
+        let interval_days = &self.days[before_days as usize..after_days as usize];
+        let intervals_and_weights = interval_days
+            .iter()
             .enumerate()
-            .collect::<Vec<_>>();
+            .map(|(interval_index, interval_day)| {
+                let sibling_multiplier = note_id
+                    .map(|note_id| {
+                        if interval_day.has_sibling(&note_id) {
+                            0.1
```

Comment:
because if all the days have siblings then we enter a world of sadness.

Reply:
Then wouldn't it be better to do something like this:
```
if all_days_have_siblings:
    sibling_multiplier = 1
else:
    sibling_multiplier = 0
```

Basically "If alldays have siblings, choose randomly, otherwise assign a 0% probability to days with siblings"

Reply:
yeah thats also reasonable, I'm not too sure it matters either way and I did change it to 0.01 in an earlier commit

Reply:
> I'm not too sure it matters

It does, because we don't want to allow siblings to appear on the same day, unless it's literally unavoidable